### PR TITLE
evgeni: rust fmt, doc cleanup, removed reth files

### PR DESCRIPTION
**Pull request by BatiGencho**
This PR formats the code with fmt, cleans up some of the docs and README and removes reth specific files which are not part of the Botanix project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,250 +1,60 @@
-## Contributing to Reth
```
Comment (arminsabouri) [inline CONTRIBUTING.md]: I think its a bit too early to change this file. Most of the advice here is relevant.

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
```
Comment (arminsabouri) [inline LICENSE]: Same thing here. I wouldn't change the licensing

File: LICENSE-APACHE [hunk 3]
```diff
@@ -1,201 +0,0 @@
-                              Apache License
```
Comment (arminsabouri) [inline LICENSE-APACHE]: Samething here

File: README.md [hunk 4]
```diff
@@ -1,146 +1,40 @@
...
+## Contracts
+Solidity contracts. Mainly the “genesis contract” the interface for peg in and outs. We use the foundry tool set for tests, building and deploying.
 
-**Modular, contributor-friendly and blazing-fast implementation of the Ethereum protocol**
+## Botanix Lib:
+A library to manage consensus changes. As a guiding principle, it is essential to recognize that minimizing the frequency of commits within Reth results in significant time savings during the process of rebasing upstream commits. 
+An example of a change to Ethereum consensus change is validating ethereum pegging attempts.
 
-![](./assets/reth-alpha.png)
+## Reth Submodule
```
Comment (arminsabouri) [inline README.md]: Maybe change this to Botanix Binary

**Review by arminsabouri — DISMISSED** [review]

**Comment by arminsabouri** [comment]
@BatiGencho In the future can you break out formatting change and other changes into different commits please

File: CONTRIBUTING.md (hunk 1 above)
Comment (BatiGencho) [reply CONTRIBUTING.md]: you are right, sorry... reverted it back

File: LICENSE (hunk 2 above)
Comment (BatiGencho) [reply LICENSE]: you are right, sorry... reverted it back

File: LICENSE-APACHE (hunk 3 above)
Comment (BatiGencho) [reply LICENSE-APACHE]: you are right, sorry... reverted it back

File: README.md (hunk 4 above)
Comment (BatiGencho) [reply README.md]: done

**Review by scottmillner — APPROVED** [review]

**Comment by stevenroose** [comment]
I'm generally against enforced code formatting on other people's code, but in the specific case here, I think it might really be a problem if you start formatting foreign code differently if you still hope to be able to apply upstream patches.

Lots of files were touched, I quickly went over the changes and it seems that almost all of the changes were in actual new botanix code, so it's not a problem. If we're touching foreign code, I'd say be careful not to introduce more merge conflicts for later.

**Pull request merged**